test/buffers_utest.cpp

```diff
@@ -80,7 +80,7 @@ class BuffersTest : public ::testing::Test {
 
     bool check_eq(const T* buf1, const T* buf2, size_t len)
     {
-        return memcmp(buf1, buf2, len * sizeof(T));
+        return (memcmp(buf1, buf2, len * sizeof(T)) == 0);
```

Comment:
outer parenthesis are useless

You can write: `return memcmp(buf1, buf2, len * sizeof(T)) == 0`